### PR TITLE
Add FAQ to Common Problems Page

### PR DESCRIPTION
**Pull request by ImmaZoni**
This PR Add forum posts tagged with FAQ as an embed onto the common problems page. This will allow for users to see forum posts on our docs without having to change webpages.

### CODE REVIEW

File: docs/protocol/common_problems.md [hunk 1]
```diff
@@ -133,3 +133,7 @@ WARN sc_service::client::client: Block import error: Potential long-range attack
 ```
 The node somehow ended up being on a fork, try wiping and starting from scratch.
 
+### Still Facing Trouble? Take a look at our forums below
+
+<iframe src="https://forum.subspace.network/search?expanded=true&q=tags%3Afaq%20order%3Alatest" width="100%" height="600px" frameborder="0" scrolling="auto"></iframe>
```
Comment (jim-counter) [inline docs/protocol/common_problems.md:138]: This does not work for me I'm afraid.

image

Comment (jim-counter) [reply docs/protocol/common_problems.md]: @ImmaZoni this now works for me with latest commit. We still need the functionality copied to latest.

**Review by jim-counter — CHANGES_REQUESTED** [review]
Needs adding to latest.

**Review by EmilFattakhov — COMMENTED** [review]
Looks good Justin, just one final adjustment before merging as per Jim's comment.

File: versioned_docs/version-latest/developers/general-information.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ The popularity of Solidity is primarily due to its design for Ethereum, the lead
 
 Lastly, Solidity is continually evolving with frequent updates, new features, and improvements that address the unique needs of blockchain development. This responsive development and the thriving community around it further solidify its position as the leading language for smart contract development.
 
-Solidity has a great community of developers and extensive documentation is available on the official [website](https://soliditylang.org/).
\ No newline at end of file
+Solidity has a great community of developers and extensive documentation is available on the official [website](https://soliditylang.org/).
```
Comment (jim-counter) [inline versioned_docs/version-latest/developers/general-information.md:49]: I can't see what the change is here?

File: versioned_docs/version-latest/developers/intro.md [hunk 3]
```diff
@@ -47,4 +47,4 @@ When archiving the history of Subspace, we replace Merkle roots with KZG commitm
 
 Our primary objective is to maintain a minimum barrier to entry for both our farmers and developers. The installation of a Subspace Network node can be accomplished in less than 15 minutes and is compatible with an extensive array of computer systems given the highly accessible minimum requirements for the hardware.
 
-When it comes to development on the Subspace Network, we offer a range of flexible options. At present, you can make use of our multiple Ethereum Virtual Machine (EVM) domains for a familiar experience. Soon, we will also provide the functionality for you to build your own local custom virtual machine if that's your preference. We take pride in the unlimited possibilities we provide - there are no boundaries!
\ No newline at end of file
+When it comes to development on the Subspace Network, we offer a range of flexible options. At present, you can make use of our multiple Ethereum Virtual Machine (EVM) domains for a familiar experience. Soon, we will also provide the functionality for you to build your own local custom virtual machine if that's your preference. We take pride in the unlimited possibilities we provide - there are no boundaries!
```
Comment (jim-counter) [inline versioned_docs/version-latest/developers/intro.md:50]: I can't see what the change is here?

**Review by jim-counter — APPROVED** [review]
There are a number of what look like unrelated changes here - wanted to make sure you're aware?

File: versioned_docs/version-latest/developers/general-information.md (hunk 2 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/developers/general-information.md:49]: This is a result of running the version-update script which brings pre-release and latest to be the same. in the two mentioned cases it was simply newlines to end the doc

File: versioned_docs/version-latest/developers/intro.md (hunk 3 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/developers/intro.md:50]: Same as above

**Pull request merged**